tests/end2end/cypress/integration/dataviz-ghaction.js

```diff
@@ -0,0 +1,44 @@
+describe('Dataviz tests', function () {
+    it('Test filtered dataviz in a popup', function () {
+        const path = require("path")
+        const PNG = require('pngjs').PNG;
+        const pixelmatch = require('pixelmatch');
+        const downloadsFolder = Cypress.config("downloadsFolder")
+
+        cy.visit('/index.php/view/map/?repository=testsrepository&project=dataviz_filtered_in_popup')
+
+        // In a perfect world, this button must be hidden, there isn't any plot to display ...
```

Comment:
the PR https://github.com/3liz/lizmap-web-client/pull/3029 will do it a perfect world ;-)

Reply:
If you want to cherry-pick this single commit in your PR, you are welcome.